radixdlt/src/main/java/com/radixdlt/consensus/BFTEventReducer.java

```diff
@@ -165,7 +176,12 @@ public void processVote(Vote vote) {
 					log.info("{}: VOTE: QC Synced: {}", this::getShortName, () -> qc);
 					synchedLog = true;
 				}
-				processQC(qc);
+				processQC(qc).ifPresent(commitMetaData -> {
+					// TODO: should this be sent by everyone and not just the constructor of the proof?
```

Comment:
Can other nodes send this? What if some nodes don't get it from this guy?

Reply:
They'll get it in other ways